cmd/e2e-runner/main.go

```diff
@@ -155,6 +155,7 @@ func realMain(ctx context.Context) error {
 		app, err := db.FindAuthorizedAppByAPIKey(deviceKey)
 		if err != nil {
 			logger.Errorf("device API key cleanup failed: %w", err)
+			return
```

Comment:
Is this best effort? Should this be a panic instead?

Reply:
Yes. Best-effort according to @yegle. I'm not sure if we want to panic - if this were a concern we could consider an async db cleanup later perhaps

Reply:
I was wondering why this log looks like:

```device API key cleanup failed: %!w(*net.OpError=&{read tcp 0xc000371b90 0xc00040f2f0 0xc0006a3760})```